src/SmartComponents/AddSystemModal/AddSystemModal.js

```diff
@@ -92,7 +117,13 @@ class AddSystemModal extends Component {
                             eventKey={ 1 }
                             title="Baselines"
                         >
-                            <BaselinesTable hasSelect={ true }/>
+                            <BaselinesTable
+                                hasMultiSelect={ true }
+                                tableType='CHECKBOX'
+                                onSelect={ this.onSelect }
+                                baselineTableData={ baselineTableData }
```

Comment:
This could now be reduced to `tableData`

Reply:
As in, rename `baselineTableData` to `tableData`?

Reply:
Yea.